src/_data/sites/hoeser.json

```diff
@@ -0,0 +1,7 @@
+{
+    "url": "https://www.hoeser.dev",
+    "name": "Raphael Höser's Blog",
+    "description": "This is the personal website of Raphael Höser (Snapstromegon)",
+    "twitter": "Snapstromegon",
+    "source_url": "https://github.com/Snapstromegon/hoeser.dev"
```

Comment:
@Snapstromegon This link is 404ing for me, so it's possibly a private repo.
We'll probably want to make the repo public or remove this optional property. ❤️

Reply:
Oh sorry, I forgot I made it private at the start. It's now public.